src/kube-runtime/test/cmd_test_job.yaml

```diff
@@ -0,0 +1,35 @@
+protocolVersion: 2
```

Comment:
do you have test code for `test/*.yaml`?

Reply:
Add test_runtime.py